messages.md

```diff
@@ -0,0 +1,62 @@
+# Messages
+
+```mermaid
+classDiagram
+    class ForgePlanningRequest {
+        +URDF
+        +SRDF
+        +CellParameters
+        +PlanningScene
+        +Feature
+        +CompositeCloud
+        +Mesh
+        +TaskParameters
+    }
+
+    class ForgePlanningResponse {
+        +TaskPlan
+    }
+
+    class GenericPlanningRequest {
+        +RobotRepresentation
+        +GroupName
+        +InitialState
+        +WorldRepresentation
```

Comment:
What objects/things do we expect to store in "WorldRepresentation" that are not already captured in "RobotRepresentation"?